### ISSUE TITLE
put dark mode localstorage

### ISSUE THREAD

**Issue by evad1n** [human37/vue3-todo-app#5]
please save

**Comment by rcorry** [comment]
please store the dark mode in RAM, should limit the number of API calls

**Comment by human37** [comment]
Good point. I think I'll do something in between, in local storage I will store the memory address of the dark mode value, and then use that address to find the actual value in ram, and use it that way 👍

**Comment by evad1n** [comment]
Will it work on my computer? Please store memory address in env variable in aws. Or maybe put memory address in .env file or make const.

Sorry sorry i read cant access memory in js. Maybe use wasm and prolog compilation to determine memory?

**Comment by evad1n** [comment]
@human37 any updates on this? This is a substantial blocker for me. @rcorry maybe you can take this?

**Comment by human37** [comment]
@evad1n sorry we have been blocked currently due to the log4j exploit
